### PR TITLE
docs: Add breaking changes for Ledger V2

### PR DESCRIPTION
**Pull request by Shiroy**
Fixes DOC-93

### CODE REVIEW

**Review by paul-nicolas — APPROVED** [review]

File: docs/ledger/reference/v2-breaking-changes.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Ledger V2 API breaking changes
+
+Even though we strive to keep the Ledger API stable, there are some changes that are necessary to make in order to implement new features or improve the existing ones. This document lists all the breaking changes that have been made to the Ledger API.
+
+## Removed Features
+
+Once you upgrade to a new version of the Ledger API, the following features will no longer be available, even if they were available in the previous version.
+
+- **Batch queries**: The ability to query multiple accounts in a single request has been removed. You must now query each account individually.
```
Comment (altitude) [inline docs/ledger/reference/v2-breaking-changes.md:13]: We should add detail on how the bulk "replaces" it imho

File: docs/ledger/reference/v2-breaking-changes.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- **Metadata format**: The ability to store metadata as a free-form JSON object on an account has been removed. You must now store metadata as a set of key-value pairs.
```
Comment (altitude) [inline docs/ledger/reference/v2-breaking-changes.md:14]: I think we could add an example

File: docs/ledger/reference/v2-breaking-changes.md (hunk 1 above)
Comment (Shiroy) [reply docs/ledger/reference/v2-breaking-changes.md:13]: J'ai pas les infos :/

Comment (Shiroy) [reply docs/ledger/reference/v2-breaking-changes.md:13]: I'll just say that it has been replaced by `bulk` and write a dedicated explanation page for bulks later

**Pull request merged**